File: LeetCode/1295. Find Numbers with Even Number of Digits/Solution.py
def num_digit(number):
    ret = 0

    while number > 0:
        ret += 1
        number //= 10

    return ret

class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        return len(list(filter(lambda l: l % 2 == 0, map(num_digit, nums))))

File: LeetCode/1295. Find Numbers with Even Number of Digits/Solution.cpp
class Solution {
private:
    int num_digit(int num) {
        int ret = 0;

        while (num > 0) {
            ret += 1;
            num /= 10;
        }

        return ret;
    }
public:
    int findNumbers(vector<int>& nums) {
        int ret = 0;

        for (int num : nums)
            ret += 1 - (num_digit(num) % 2);
        return ret;
    }
};
